File: Codechef/Beginner/LuckyFour.java
import java.util.Scanner;

public class LuckyFour {
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		int numberOfElements = scan.nextInt();
		while(numberOfElements > 0) {
			int number = scan.nextInt();
			int count = 0;
			while(number > 0) {
				if((number % 10) == 4) {
					count++;
				}
				number /= 10;
			}
			System.out.println(count);
			numberOfElements--;
		}
		scan.close();
	}
}

File: Codechef/Python/LuckyFour.py
numberOfTest = int(input())
while(numberOfTest > 0):
	count = 0
	number = int(input())
	while(number > 0):
		test = number % 10
		if(test == 4):
			count += 1
		number /= 10
	print(count)
	numberOfTest -= 1
